src/test/java/ru/yandex/practicum/filmorate/controller/FilmControllerTest.java

```diff
@@ -73,8 +90,8 @@ public void shouldThrowExceptionIfFilmWithNotValidReleaseDate() {
                 .duration(100)
                 .releaseDate(LocalDate.of(1600, 6, 1))
                 .build();
-        ValidationException exception = assertThrows(ValidationException.class, () -> {controller.add(film);});
-        assertEquals("Дата релиза фильма не может быть раньше " + controller.FIRST_FILM_RELEASE_DATE,
+        ValidationException exception = assertThrows(ValidationException.class, () -> {controller.create(film);});
+        assertEquals("Дата релиза фильма не может быть раньше " + filmService.FIRST_FILM_RELEASE_DATE,
```

Comment:
К статичным полям можно получать доступ по названию класса:
```suggestion
        assertEquals("Дата релиза фильма не может быть раньше " + FilmService.FIRST_FILM_RELEASE_DATE,
```

Reply:
Done